### DISCUSSION TITLE
Message is delivered between android app client when apps are live and connected to server web socket.

### DISCUSSION THREAD

**Discussion by GitHub-Ram** [indrawp/Setup-Guide#98] [Q&A]
Hi,
I am running server(v5.9x) and android client(v5.12.x) with current master build.
Registration of the app client on server and every thing look fine.
Log show : RegistrationUtil: Marking registration completed.
We made two app user registered.
But while sending message between that two client , server receive message from sender but not being received by destination.
Message Log like this which is being send by one of the app.

request {
body: "{"destination":"65aa882a-fa47-4cdb-a7a3-7a170143a471","messages":[{"content":"MwigjJIFEiEF7w2CwebJbeLW401FsO2x0/FXECw+VrQPHv+F2XsYKj4aIQX0F5TBvioakYs6EeuwQUAcvmdwh2II8FDWrg39WSdZNSLTATMKIQV39oIJOXlL80T9qguUl6YzAtSRHrok+pE/iPmZfq8JVhAAGAAioAH5RcDtaUtVGq5PSRItujgEPOhGjWShwakU3TxFfajP+mlFfCYcduxnKv3SvkVlqV+8d58GxUyUUXl8xyLqqKl98kZ8nz5PITzAVjQobDmo6chJhKXDA6dRHMZiXLtKUJWZoygyML8tLCBf7NMouIHp9eyKxU2vScm9Ty0eByTqhdOdQDVIpCy72nafUdKXnWurqlDcY65QNoC70NYcF9qxEEV5pJUDmgMoh04w/ffZAg==","destinationDeviceId":1,"destinationRegistrationId":1803,"type":3}],"online":false,"timestamp":1621931404083}"
headers: "content-type:application/json"
id: 4863838312246111685
path: "/v1/messages/65aa882a-fa47-4cdb-a7a3-7a170143a471"
verb: "PUT"
}

Also received 200 from server.
But destination do not receive any thing on web-socket as long as app and socket connection is live.
While we relaunch the app or restart the server, all those messages send are received on by the app client.
Please suggest me probable configuration that I might have done wrong.
Please guid me how to configure Redis DB and Dynamo DB. 
I don't understand Pub/Sub mechanism used. 
Please suggest me how to solve such issue.

**Comment by madeindra** [comment]
Yeah this is a common issue, I will inform once I found the answer, but for now you can reopen the app on the client

**Comment by ikblmul** [comment]
Hey I found solution for this You need enable config set notify-keyspace-events KEA in Redis cli, I'm checkout this in file signal app clientPrescentManager.java
